src/codegen/browser/test.ts

```diff
@@ -137,7 +137,7 @@ function buildBrowserNodeGraph(events: BrowserEvent[]) {
           modifiers: event.modifiers,
           inputs: {
             previous,
-            locator: getLocator(event.tab, event.selector),
+            locator: getLocator(event.tab, event.target.selectors),
```

Comment:
`LocatorNode` also needs a similar change, but I would rather do it in a separate PR